### PR TITLE
Add VPA and region fix

### PR DESCRIPTION
**Pull request by paurosello**
- **This repository allows merge commits, if you're creating a Release, make sure to select 'Squash and merge'**
- **When merging subtree updates, it is important to select 'Create a merge commit' when merging.**

This PR..

### CODE REVIEW

File: helm/aws-load-balancer-controller/templates/vpa.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      controlledValues: RequestsAndLimits
+      mode: Auto
+  targetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name:  {{ include "aws-load-balancer-controller.fullname" . }}
+  updatePolicy:
+    updateMode: Auto
+{{ end }}
+{{ end }}
```
Comment (Gacko) [inline helm/aws-load-balancer-controller/templates/vpa.yaml:23]:
```suggestion
{{ end }}
```

**Review by Gacko — CHANGES_REQUESTED** [review]
Please regenerate the `values.schema.json`. Everything else LGTM! :)

File: helm/aws-load-balancer-controller/templates/vpa.yaml (hunk 1 above)
Comment (paurosello) [reply helm/aws-load-balancer-controller/templates/vpa.yaml:21]: when regenerating schema values all hell breaks loose as many fields are set to Null

**Comment by paurosello** [comment]
not doing the schema update as it will mess a bunch of fields now

**Review by T-Kukawka — APPROVED** [review]

**Review by pipo02mix — APPROVED** [review]
LGTM though I did not test it

**Comment by LolloneS** [comment]
I fear the installation could fail because of the "wrong" schema, is it tested?

**Comment by paurosello** [comment]
The schema did not change, I did not add/modify any value on the values.yaml file so I expect it to work, doing some tests anyway

**Pull request merged**